.ko.yaml

```diff
@@ -2,3 +2,4 @@
 defaultBaseImage: gcr.io/distroless/static:nonroot
 baseImageOverrides:
   knative.dev/serving/vendor/github.com/tsenart/vegeta: ubuntu:latest
+  knative.dev/serving/test/test_images/hellovolume: ubuntu:latest
```

Comment:
Why do we need this?  Can you add a comment?

Reply:
I think this is debug. And I think there was a question about it already :D

Reply:
@mattmoor sure I will add. Without a different image I will a permission issue as described in my [comment ](https://github.com/knative/serving/pull/9264#issuecomment-686564258) above.

Reply:
@vagababov could you elaborate a bit more about the bug? The same question was asked in the previous PR.

Reply:
what happens if you do an explicit `runAsUser: root` with the distroless image?

Reply:
@julz I thought about it, but feels kind of awkward to use a `nonroot` image to run things as root.

Reply:
FWIW my personal opinion would be that - if it works! - explicitly using `runAsUser: root` might be a bit nicer (and clearer) than using an entirely different image just for the side-effect of its default `user` happening to be root. Apart from anything else nothing technically stops `ubuntu:latest` changing to a non-root user at some point and breaking the test! 

Worth a try anyway to see if it's an option.

Reply:
@julz it worked so we are ready to go :)